src/routes/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { BrowserRouter, Route, Switch} from 'react-router-dom'
+import Home from '../components/Home'
+import Favorites from '../components/Favorites'
+
+export default () => {
+  return (
+    <BrowserRouter>
+      <Switch>
+        <Route exact path="/" component={Home}/>
+        <Route exact path="/favorites" component={Favorites}/>
+      </Switch>
```

Comment:
It is a good idea to have a redirect to avoid unknown routes, you an either redirect to an existing route with a message or use a not found route